MekHQ/src/mekhq/campaign/parts/Refit.java

```diff
@@ -902,6 +902,9 @@ public void calculate() {
             }
             if (oldHS != newHS) {
                 Part hsPart = heatSinkPart(newEntity); // only single HS allowed, so they have to be of the same type
+                if (isOmniRefit) {
+                    hsPart.setOmniPodded(true);
+                }
```

Comment:
Couldn't we just set this to omniRefit and ignore the if statement?

Reply:
Unless I'm missing something, the block starting a L904 is reached by any vehicle refit that changes the number of external (non-engine-free) heat sinks. (i.e. not just OmniVehicles)

Reply:
Correct, but is the setOmniPodded to isOmniRefit cheaper than an if statement followed the setter?

Reply:
Ah, I understand what you're saying  now - good suggestion.